app/src/main/java/com/andrew/tasky/agenda/data/event/EventCompressor.kt

```diff
@@ -0,0 +1,61 @@
+package com.andrew.tasky.agenda.data.event
+
+import android.content.Context
+import com.andrew.tasky.agenda.data.storage.ImageStorage
+import com.andrew.tasky.agenda.data.util.BitmapConverters
+import com.andrew.tasky.agenda.domain.models.AgendaItem
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.async
+import kotlinx.coroutines.supervisorScope
+import kotlinx.coroutines.withContext
+
+class EventCompressor(context: Context) {
```

Comment:
Aren't it rather the event's photos that are compressed and not the event itself?